### PR TITLE
HEHA-193 Update HEHA! prototype

### PR DESCRIPTION
**Pull request by jackdcrawford**
#### What does this PR do? (please provide any background)
Adds an `example/` directory with the markup for HEHA! planner from `tripapplite` in its current state, ready for [No Divide](http://nodividestudio.com/) to pick it up and apply a redesign next week.

#### What tests does this PR have?
None

#### How can this be tested?
Please follow the new `README.md` and make sure everything makes sense. Try out the *Workflow* section locally.

#### Screenshots / Screencast
heha prototype

#### What gif best describes how you feel about this work?

---

- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by Joezo** [comment]
Taking a review

**Comment by Joezo** [comment]
Looking good!

**Comment by jodiedoubleday** [comment]
:shipit:

File: README.md [hunk 1]
```diff
@@ -1,86 +1,65 @@
...
-1. Increase the build's patch number ( e.g. v 1.2.__3__ => v 1.2.__4__ )
-2. Build & Package Distribution Files
-3. Create Tag Branch & Perform a git commit
-4. Push `dist` files to AWS S3 bucket listed in `aws.json`
+Watch for changes in `src/` and `example/theme.less` & recompile
+```bash
+npm run example:watch
+```
 
+Monitor files for changes and reload your web browser with [livereload]([livereload](https://www.npmjs.com/package/livereload)
```
Comment (hxmattobee) [inline README.md]: I think this should be: `[livereload](https://www.npmjs.com/package/livereload)`

**Pull request merged**